src/parser-includes.ts

```diff
@@ -104,6 +104,23 @@ export class ParserIncludes {
 
                     includeDatas = includeDatas.concat(await this.init(fileDoc, opts));
                 }
+            } else if (value["component"]) {
+                const {domain, projectPath, componentName, ref} = this.parseIncludeComponent(value["component"]);
+                // converts component to project
```

Comment:
Btw, not sure if this is a good idea

And not sure if I am missing out anything , but include:component seemed to be just include:project 

not sure what additional feats include:component provides